simulators/kcc/deploy-contract/main.go

```diff
@@ -0,0 +1,167 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/ethereum/hive/hivesim"
+	Store "github.com/ethereum/hive/simulators/kcc/deploy-contract/contract"
+	"math/big"
+	"strconv"
+	"time"
+)
+
+func main() {
+	suite := hivesim.Suite{
+		Name:        "kcc-deploy-contract",
+		Description: "A general deploy-contract test case for KCC",
+	}
+	suite.Add(hivesim.ClientTestSpec{
+		Role:        "eth1",
+		Name:        "mine one block",
+		Description: "Waits for a single block to be mined by kcc",
```

Comment:
@newgoo  

I think you should change the `name` and `description` of the test case.

Reply:
fixed